config/routes.rb

```diff
@@ -2,150 +2,152 @@
+    if Figaro.env.enable_test_routes == 'true'
+      namespace :test do
+        # Assertion granting test start + return.
+        get '/saml' => 'saml_test#start'
+        get '/saml/decode_assertion' => 'saml_test#start'
+        post '/saml/decode_assertion' => 'saml_test#decode_response'
+        post '/saml/decode_slo_request' => 'saml_test#decode_slo_request'
+      end
     end
-  end
 
-  # Non-devise-controller routes. Alphabetically sorted.
-  get '/.well-known/openid-configuration' => 'openid_connect/configuration#index',
-      as: :openid_connect_configuration
-
-  get '/account' => 'accounts#show'
-  get '/account/reactivate/start' => 'reactivate_account#index', as: :reactivate_account
-  put '/account/reactivate/start' => 'reactivate_account#update'
-  get '/account/reactivate/verify_password' => 'users/verify_password#new', as: :verify_password
-  put '/account/reactivate/verify_password' => 'users/verify_password#update', as: :update_verify_password
-  get '/account/reactivate/verify_personal_key' => 'users/verify_personal_key#new',
-      as: :verify_personal_key
-  post '/account/reactivate/verify_personal_key' => 'users/verify_personal_key#create',
-       as: :create_verify_personal_key
-  get '/account/verify_phone' => 'users/verify_profile_phone#index', as: :verify_profile_phone
-  post '/account/verify_phone' => 'users/verify_profile_phone#create'
-
-  get '/api/health/workers' => 'health/workers#index'
-  get '/api/openid_connect/certs' => 'openid_connect/certs#index'
-  post '/api/openid_connect/token' => 'openid_connect/token#create'
-  match '/api/openid_connect/token' => 'openid_connect/token#options', via: :options
-  get '/api/openid_connect/userinfo' => 'openid_connect/user_info#show'
-  get '/api/saml/metadata' => 'saml_idp#metadata'
-  match '/api/saml/logout' => 'saml_idp#logout',
-        via: %i[get post delete],
-        as: :destroy_user_session
-  match '/api/saml/auth' => 'saml_idp#auth', via: %i[get post]
-
-  post '/api/service_provider' => 'service_provider#update'
-
-  delete '/authenticator_setup' => 'users/totp_setup#disable', as: :disable_totp
-  get '/authenticator_setup' => 'users/totp_setup#new'
-  patch '/authenticator_setup' => 'users/totp_setup#confirm'
-  get '/authenticator_start' => 'users/totp_setup#start'
-
-  get '/forgot_password' => 'forgot_password#show'
-
-  get '/manage/email' => 'users/emails#edit'
-  match '/manage/email' => 'users/emails#update', via: %i[patch put]
-  get '/manage/password' => 'users/passwords#edit'
-  patch '/manage/password' => 'users/passwords#update'
-  get '/manage/phone' => 'users/phones#edit'
-  match '/manage/phone' => 'users/phones#update', via: %i[patch put]
-  get '/manage/personal_key' => 'users/personal_keys#show', as: :manage_personal_key
-  post '/manage/personal_key' => 'users/personal_keys#update'
-
-  get '/openid_connect/authorize' => 'openid_connect/authorization#index'
```

Comment:
Ditto for the `/openid_connect` URLs (or anything we publish in the developer docs, those should not be locale-scoped IMO